src/pairings.rs

```diff
@@ -210,9 +210,6 @@ impl<'b> AddAssign<&'b MillerLoopResult> for MillerLoopResult {
 #[derive(Copy, Clone, Debug)]
 pub struct Gt(pub(crate) Fp12);
 
-#[cfg(feature = "zeroize")]
-impl zeroize::DefaultIsZeroes for Gt {}
```

Comment:
was this just a merge artifact? or why do we have multiple definitions?

Reply:
We originally did this in our fork but they added it in upstream between 0.7 and 0.8